Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -575,7 +576,7 @@ public extension String {
 	/// - Parameter string: substring to search for.
 	/// - Returns: first index of substring in string (if applicable).
 	public func firstIndex(of string: String) -> Int? {
```

Comment:
This is probably obsolete now that `String` is a `Collection`?
We can use `func index(of element: Self.Element) -> Self.Index?` I think?